bundles/org.openhab.binding.opensprinkler/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -96,4 +97,11 @@
 		<description>Controls a station on the OpenSprinkler device.</description>
 		<category>Switch</category>
 	</channel-type>
+	
+	<channel-type id="remainingWaterTime">
+		<item-type>Number:Time</item-type>
+		<label>Remaining Water Time</label>
+		<description>Read-only property of the remaining water time of the station.</description>
+		<state readOnly="true" pattern="%.0f min"/>
```

Comment:
I think minutes is the most used value, so having it as a default seems to make sense? :)

Reply:
Probably. Did you check that this works?

Reply:
Yep, tested it and it works like a charme :)